addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3251,17 +3271,17 @@ var SnippetsMenu = Widget.extend({
                             if (temp[k]['drop-in']) {
                                 $selectorChildren = $selectorChildren.add(temp[k]['drop-in'].all());
                             }
+                            if (temp[k]['drop-exclude-ancestor']) {
+                                selectorExcludeParent.push(temp[k]['drop-exclude-ancestor']);
+                            }
```

Comment:
Can you explain me why we don't need that for `data-drop-lock-within` :thinking:

Reply:
Because `data-drop-lock-within` is about being locked within a parent - but that part is the code about being dragged from the Blocks list - where there is no "in page" parent yet.

Reply:
All clear thanks :+1: